tracing/opentelemetry-tracing-sync/src/main/scala/sttp/tapir/server/opentelemetry/package.scala

```diff
@@ -0,0 +1,5 @@
+package sttp.tapir.server
+
+package object opentelemetry {
+  type Identity[A] = A  // Type alias pour le style synchrone

```

Comment:
this is already available in sttp.shared

Reply:
Thanks for catching this. Indeed, I'll remove this type alias that's already available in sttp.shared and use the proper import instead.

I'll push a commit with this change.